### PR TITLE
update v1.16.0

### PR DESCRIPTION
**Pull request by dreamerlxb**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     </parent>
     <groupId>org.webjars</groupId>
     <artifactId>bootstrap-table</artifactId>
-    <version>1.15.6-SNAPSHOT</version>
+    <version>1.16.0</version>
```
Comment (jamesward) [inline pom.xml]: This needs to be `1.16.0-SNAPSHOT` because Maven performs the actual release.

Comment (dreamerlxb) [reply pom.xml]: ok，updated!

**Comment by jamesward** [comment]
Don't need this anymore?

**Comment by mwxp06** [comment]
@jamesward 
Please release new version 1.16.0

**Comment by jamesward** [comment]
This will be released shortly.

**Pull request merged**